README.md

```diff
@@ -1,46 +1,119 @@
-in-toto java
-============
+# in-toto java
 
 This repository contains an in-toto compliant library in Java. This document
 describes the repository layout, the usage purpose of this library as well as
 its current limitations.
 
 # Usage
 
-## installation
+## Installation
 
-This library is intended to be used with maven buildsystem, although you can
-probably easily move it to any other if you're familiar with those. To add it
-to your mvn project edit the pom.xml file to add:
+This library is intended to be used with maven build system, although you can
+probably easily move it to any other if you're familiar with those. To add it to
+your mvn project edit the pom.xml file to add:
 
 ```xml
     ...
     <dependency>
-      <groupId>io.github.in-toto</groupId>
-      <artifactId>in-toto</artifactId>
-      <version>0.1</version>
-      <scope>compile</scope>
+    <groupId>io.github.in-toto</groupId>
+    <artifactId>in-toto</artifactId>
+    <version>0.3.3</version>
     </dependency>
     ...
 ```
 
-With it you should be able to use the library inside of your project.
+With it, you should be able to use the library inside your project.
 
-## Using the library
+## Using the new library
 
-The library exposes a series of objects and convenience methods to create,
-sign, and serialize in-toto metadata. As of now, only Link metadata is
-supported (see the Limitations section to see what exactly is supported as of
-now).
+The library exposes a new set of models used for in-toto 0.1.0, DSSE 1.0.0 and .
+If you wish to use the deprecated legacy Link library, please skip to the next
+section.
 
-Metadata classes are located in the `io.in_toto.models.*` namespace. You can,
-for example create a link as follows:
+The new library allows you to instantiate a Statement and populate it as
+follows:
 
 ```java
-    Link link = new Link(null, null, "test", null, null);
+Subject subject=new Subject();
+subject.setName("curl-7.72.0.tar.bz2");
+subject.setDigest(
+Map.of(
+DigestSetAlgorithmType.SHA256.toString(),
+"d4d5899a3868fbb6ae1856c3e55a32ce35913de3956d1973caccd37bd0174fa2"))
+Predicate predicate=createPredicate();
+Statement statement=new Statement();
```

Comment:
optional: create a version of the constructor that accepts the subject list and predicate as args so users don't have to set them manually.